packages/typescript-resolver-files/src/validatePresetConfig/validatePresetConfig.ts

```diff
@@ -242,6 +245,7 @@ export const validatePresetConfig = ({
     typesPluginsConfig: validatedTypesPluginsConfig,
```

Comment:
Since we are using the preset config's `emitLegacyCommonJSImports`, I think we should throw an error in [validateTypesPuginConfig](https://github.com/eddeee888/graphql-code-generator-plugins/blob/cfa99e22c4bf1347eec6ca834e838626404c4a22/packages/typescript-resolver-files/src/validatePresetConfig/validatePresetConfig.ts#L248-L265) if `typesPluginsConfig.emitLegacyCommonJSImports` is set.